app/services/invoices/customer_usage_service.rb

```diff
@@ -2,20 +2,19 @@
 
 module Invoices
   class CustomerUsageService < BaseService
+    # NOTE: customer_id can reference the lago id or the external id of the customer
+    # NOTE: subscription_id can reference the lago id or the external id of the subscription
```

Comment:
Merf :/
Something that we might improve in a near future :smile:

Reply:
Yes - That's why I put a note here 😅